draft-ietf-quic-load-balancers.md

```diff
@@ -902,19 +915,29 @@ A QUIC version might define limits on connection ID length that make some or all
 of the mechanisms in this document unusable.  For example, a maximum connection
 ID length could be below the minimum necessary to use all or part of this
 specification; or, the minimum connection ID length could be larger than the
-largest value in this specification.
+largest value in this specification. Similarly, the length self-encoding
+specification cannot accommodate connection IDs longer than 32 bytes.
+
+The advanced fallback implementation supports a requirement to inspect version-
+specific elements of packets to make a routing decision, such as the Server Name
+Indication (SNI) extension in the TLS Client Hello.  The format and
+cryptographic protection of this information may change in future versions or
+extensions of TLS or QUIC, and therefore this functionality is inherently
+version-dependent. Such a load balancer, when it receives packets from an
+unknown QUIC version, might misdirect initial packets to the wrong tenant. While
```

Comment:
lowercase "initial" is not "Initial" obviously because uppercase implies something specific.  However, it might be better to just drop that word.